build_tools/cmake/rebuild.sh

```diff
@@ -27,20 +27,21 @@ CMAKE_BIN=${CMAKE_BIN:-$(which cmake)}
 ninja --version
 
 cd ${ROOT_DIR?}
-if [ -d "build" ] 
+if [ -d "build" ]
 then
-  echo "Build directory already exists. Will use cached results there." 
+  echo "Build directory already exists. Will use cached results there."
 else
   echo "Build directory does not already exist. Creating a new one."
   mkdir build
 fi
 cd build
 "$CMAKE_BIN" -G Ninja -DCMAKE_BUILD_TYPE=FastBuild \
+                      -DCMAKE_C_COMPILER=clang \
+                      -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
Delete these. Otherwise you're not using the env variable. If you want to make this script work without the env variables specified, you can add

```shell
CC="${CC:-$(which clang)}"
CXX="${CXX:-$(which clang++)}"
```

to have it default to these if the env variable isn't specified. That shouldn't be necessary for this PR though and I can do it in a followup.